src/core/ftd-shared.cmake

```diff
@@ -0,0 +1,60 @@
+#
```

Comment:
it seems this is for posix platform only. If so, can we move the CMake file into `src/posix`? You may use get_target_property() to retrieve SOURCES and INCLUDE_DIRECTORIES of openthread-ftd and openthread-mtd.

Reply:
Updated